.github/workflows/ci-test.yml

```diff
@@ -139,3 +139,19 @@ jobs:
         env:
           TARGET_BRANCH: ${{ needs.prepare.outputs.changes-target-branch }}
       - uses: ./.github/actions/build-teardown
+
+  e2e-test:
+    runs-on: ubuntu-latest-xl
+    timeout-minutes: 70
+    continue-on-error: false
+    needs: prepare
+    container:
+      image: libra/build_environment:github-1
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - uses: ./.github/actions/build-setup
+      - name: run e2e tests
+        run: cargo x test --package smoke-test -- --test-threads 2 --nocapture
```

Comment:
this is fine for now, but we'll need the 3x retries.

Reply:
Yeah I was going for simple for now (although I haven't seen any failures yet)